test/src/tests-identities-attributes.ts

```diff
@@ -799,11 +800,12 @@ describe('identities and attributes', function() {
      });
 
     it('should get cart products', function(done) {
-        const product1 = mParticle.eCommerce.createProduct('iPhone', 'SKU1', 1),
-            product2 = mParticle.eCommerce.createProduct('Android', 'SKU2', 1);
+        const product1: SDKProduct = mParticle.eCommerce.createProduct('iPhone', 'SKU1', 1, 1);
+        const product2: SDKProduct = mParticle.eCommerce.createProduct('Android', 'SKU2', 1, 1);
+
         waitForCondition(hasIdentifyReturned)
         .then(() =>  {
-        mParticle.eCommerce.Cart.add([product1, product2], false);
+        mParticle.eCommerce.Cart.add([product1, product2], null);
```

Comment:
this shouldn't matter since it's deprecated, but i believe the 2nd argument shoudl be true/false bc it indicated if it should log the event or not.  but this test will be removed in the future...so nbd